templates/app/views/layouts/storefront.html.erb.tt

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html lang="<%%= I18n.locale %>">
+  <head>
+    <meta charset="utf-8">
+    <title><%%= title %></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <%%= meta_data_tags %>
```

Comment:
Are those `<%%=` typos from `<%=`?

Reply:
No, that's a way to let the `.tt` file render the erb tags without processing them. Close to the `importmap` directive, you can see an occurrence of a tag that is instead interpreted and won't be shown in the view on the host application.

Reply:
Oh, I see. Thanks.